src/Validator/Rules/ValidationRule.php

```diff
@@ -15,7 +15,7 @@ abstract class ValidationRule
 
     public function getName()
     {
-        return $this->name ?: static::class;
+        return $this->name !== '' ? $this->name : static::class;
```

Comment:
Can we invert the condition to avoid negation?

Reply:
4a6ffb4